### PR TITLE
fix: [OIP-676]: Add connector summary details in the connector page for AppD, ELK and Prometheus

### PR DESCRIPTION
**Pull request by praneshg239**
#### Screenshots

Overview
Overview Referenced by Activity History
Pasted Graphic 15

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/35-connectors/pages/connectors/__tests__/mockData.ts [hunk 1]
```diff
@@ -954,3 +954,69 @@ export const SignalFXConnectorDetails: ConnectorInfoDTO = {
     delegateSelectors: ['qa-automation-cv-chiplay']
   }
 }
+
+export const ELKConnectorDetails = {
+  name: 'elk mtls',
+  identifier: 'elk_mtls',
+  description: '',
+  accountIdentifier: '-k53qRQAQ1O7DBLb9ACnjQ',
```
Comment (mayankVermaHarness) [inline src/modules/35-connectors/pages/connectors/__tests__/mockData.ts:962]: Please try to add dummy AccountIds in mocks

**Review by mayankVermaHarness — APPROVED** [review]

File: src/modules/35-connectors/pages/connectors/__tests__/mockData.ts (hunk 1 above)
Comment (praneshg239) [reply src/modules/35-connectors/pages/connectors/__tests__/mockData.ts]: Updated

**Review by mayankVermaHarness — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

File: src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx [hunk 2]
```diff
@@ -957,6 +957,81 @@ const getSignalFXSchema = (connector: ConnectorInfoDTO): Array<ActivityDetailsRo
...
+    {
+      label: 'UrlLabel',
+      value: connector?.spec?.url
+    },
+    {
+      label: 'username',
+      value: connector?.spec?.username
+    },
+    {
+      label: 'connectors.jenkins.passwordAPIToken',
```
Comment (n00bitax) [inline src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx:1000]: jenkins ?

Comment (praneshg239) [reply src/modules/35-connectors/pages/connectors/views/savedDetailsView/SavedConnectorDetails.tsx:1000]: yes, this is a display label

**Pull request merged**